viewer/vue-client/src/components/layout/navbar.vue

```diff
@@ -7,23 +9,27 @@ export default {
   data() {
     return {
       hasAvatar: true,
-      tabs: [
-        { id: 'Search', text: 'Search', link: '/search/libris' },
-        { id: 'Create new', text: 'Create new', link: '/create' },
-        { id: 'Directory care', text: 'Directory care', link: '/directory-care' }, 
-      ],
     };
   },
   components: {
     'user-avatar': UserAvatar,
     'tab-menu': TabMenu,
   },
   computed: {
-    user() {
-      return this.$store.getters.user;
-    },
-    settings() {
-      return this.$store.getters.settings;
+    ...mapGetters([
+      'userCare',
+      'settings',
+      'user',
+    ]),
+    tabs() {
+      const $directoryCareBadge = this.userCare.length === 0 ? '' : `<span class="badge badge-accent">${this.userCare.length}</badge>`;
```

Comment:
How about lifting badge up a bit?

```suggestion
      const $directoryCareBadge = this.userCare.length === 0 ? '' : `<span class="badge badge-accent" style="position:relative; top:-6px;">${this.userCare.length}</badge>`;
```

Reply:
Haven't tried. Can you post a screenshot?

Reply:
![after](https://user-images.githubusercontent.com/22232928/54018986-517dc500-418a-11e9-8245-4a3c8e56514e.png)
:)

Reply:
IMO the badge should align with the rest of the line.

https://getbootstrap.com/docs/3.3/components/#Badges

If we want it to have more of a "bubble" look, we can align it up a bit but then we should also decrease the size.

Reply:
Ok fair enough.